qgis-samgeo-plugin/deps_install_dialog.py

```
@@ -0,0 +1,296 @@
+                )
+        except Exception:
+            if sys.platform == "darwin":
+                self.gpu_label.setText(
+                    "Could not detect NVIDIA GPU.\n"
+                    "CUDA will not be enabled. Apple Silicon MPS may still be "
+                    "used at runtime if available."
+                )
+            else:
+                self.gpu_label.setText("Could not detect GPU.\nCPU mode will be used.")
+
+    def _on_reinstall_clicked(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_on_reinstall_clicked`

Add return type annotation: `None`

(ANN202)